nix/tictactoe.nix

```diff
@@ -0,0 +1,44 @@
+{ self, pkgs, ... }:
+
+
+let
+  stechec2-src = fetchGit {
+    url = "https://github.com/prologin/stechec2.git";
+    rev = "12a087716795e965354aac57904ee840f65fe789";
+  };
+
+  stechecWithGame = pkgs.stdenv.mkDerivation {
+    name = "stechecWithGame";
+    version = "1.0";
+
+    src = ../games/tictactoe;
+
+    builder = pkgs.writeShellScript "builder" ''
+      export PATH="${pkgs.coreutils}/bin"
+      cp --no-preserve=mode,ownership -r ${stechec2-src} $out
+      cp --no-preserve=mode,ownership -r $src $out/games/tictactoe
+    '';
+  };
+in
+
+pkgs.stdenv.mkDerivation {
+  name = "tictactoe";
+  version = "1.0";
+
+  src = stechecWithGame;
+
+  buildInputs = with pkgs; [
```

Comment:
pkg-config and wafHook should likely be nativeBuildInputs. gtest/gcovr likely checkInputs. Not sure why gcc needs to be in there, it's part of the stdenv, if it's actually needed in buildInputs that's likely hiding a serious bug (e.g. using the wrong compiler when building in certain environments).

(Similar comment for the deps list in the main stechec2 derivation -- these two should be deduped somehow.)